bids2openminds/converter.py

```diff
@@ -1,23 +1,28 @@
 from warnings import warn
 from bids import BIDSLayout, BIDSValidator
 import os
+import click
 from . import main
 from . import utility
 from . import globals
 
 
+@click.command()
+@click.argument("input_path", type=click.Path(file_okay=False,exists=True))
+@click.option("-o","--output_path",default=None,type=click.Path(file_okay=True,writable=True),help="The output path or filename for OpenMINDS file/files.")
+@click.option("--single-file","output_type",flag_value=True,default=False,help="Save the entire collection into a single file (default).")
```

Comment:
`output_type` is very unclear as an argument name, would be better as `output_separate_files` or (with inverted meaning) `output_single_file`.